src/Xamarin.Android.Build.Tasks/Tasks/CopyResource.cs

```diff
@@ -19,66 +18,29 @@ public class CopyResource : Task
 		[Required]
 		public string OutputPath { get; set; }
 
-		static readonly Assembly assm = Assembly.GetExecutingAssembly ();
+		static readonly Assembly ExecutingAssembly = Assembly.GetExecutingAssembly ();
+		static readonly Assembly JcwGenAssembly = typeof (JavaCallableWrapperGenerator).Assembly;
 
 		public override bool Execute ()
 		{
-			return Run (assm, ResourceName, OutputPath, Log);
-		}
-
-		public bool Run (Assembly assm, string ResourceName, string OutputPath, TaskLoggingHelper Log)
-		{
-			// Ensure our output directory exists
-			if (!Directory.Exists (Path.GetDirectoryName (OutputPath)))
-				Directory.CreateDirectory (Path.GetDirectoryName (OutputPath));
```

Comment:
NOTE: we don't need this because `CopyIfStreamChanged` (and other variants) check if the directory exists.